src/open_inwoner/pdc/tests/test_category_admin.py

```diff
@@ -95,6 +96,52 @@ def test_user_cannot_publish_child_category_without_root_published_on_list_page(
         updated_category = Category.objects.get(slug="bar4")
         self.assertFalse(updated_category.published)
 
+    def test_access_limited_to_linked_auth_groups(self):
```

Comment:
I think we're missing a testcase to verify that a user with managed categories cannot see/edit the `access_groups` field on the `Category` change page

Reply:
I added this test (even though it is now readonly)